tools/clang/test/HLSLFileCheck/hlsl/types/conversions/attributes_Mod.hlsl

```diff
@@ -177,9 +177,6 @@ Texture2D<float4> tex1[10] : register( t20, space10 );
   |-RegisterAssignment <col:30> register(t20, space10)
 */
 
-[outputcontrolpoints(-1)] // expected-warning {{attribute 'outputcontrolpoints' must have a uint literal argument}} fxc-pass {{}}
-void all_wrong() { }
```

Comment:
This is actually a FileCheck test, not a verify test, so this expect wasn't checked before and replacing it with an error results in no output to FileCheck so the whole test fails. But now that I look at it again, there's only one not-so-meaningful CHECK so this test should probably be fixed in a follow up... I don't think I can quickly fix it here since I'm not familiar with how any of the `fxc` checks are meant to be run. Filed #6359.